src/main/java/duke/ui/TextUi.java

```diff
@@ -0,0 +1,23 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+public class TextUi {
+    private final Scanner in;
+    private static int numberOfInput = 0;
```

Comment:
Should this be named numberOfInputs?

Reply:
Hello, thank you for the inputs. I've made the relevant changes!